ckanext/ontario_theme/templates/external/scheming/package/read.html

```diff
@@ -29,31 +28,33 @@ <h2>{% trans %}For more information{% endtrans %} </h2>
     {% endif %}
   {% endif %}
 
-  <div>
-    {% if contact_name %}  
-      {{contact_name}}
-    {% endif %}
-    {% if contact_name and contact_email %}
-      &nbsp;|&nbsp;
-    {% endif %}
-    {% if contact_page %}
+  <div class="ministry-contact">
+	<b>{% trans %}Ministry contact: {% endtrans %} </b>
+	{% if contact_email %}
+	  {% if contact_name %}
+      <a href="mailto:{{contact_email}}?subject={{pkg.title}}">
+        {{contact_name}}
+      </a>
+	  {% endif %}
```

Comment:
can combine the two `if` statements into one and just use `else` to distinguish if name or no name.